spec/features/admin/admin_sees_order_details_spec.rb

```diff
@@ -0,0 +1,82 @@
+                         password: "Seekrit",
+                         full_name: "Sir Whats",
+                         display_name: "mrwhatsalot")
+
+    @admin = User.create!(email: "admin@example.com",
+                          password: "password",
+                          full_name: "jeff admin",
+                          display_name: "admin",
+                          role: 1)
+
+    @item = Item.create!(title: "Eat Cheese and Die",
+                         description: "The spookiest grilled cheese.",
+                         price: 50.65,
+                         categories: [category])
+
+    @item2 =  Item.create!(title: "It Came From Beneath the Sink",
+                           description: "You don't want to know.",
+                           price: 200.99,
+                           categories: [category])
+
+    @order = Order.create!(user_id: @user.id,
+                           total_price: 50.65,
+                           items: [@item],
+                           created_at: "2015-06-11 19:35:07",
+                           updated_at: "2015-06-13 10:22:17",
+                           status: "ordered")
+
+    @order2 = Order.create!(user_id: @user.id,
+                            total_price: 89.95,
+                            items: [@item, @item2],
+                            created_at: "2015-06-10 19:35:07",
+                            updated_at: "2015-06-12 10:22:17",
+                            status: "complete")
+
+    @order3 = Order.create!(user_id: @user.id,
+                            total_price: 89.95,
+                            items: [@item],
+                            created_at: "2015-06-09 19:35:07",
+                            updated_at: "2015-06-23 10:22:17",
+                            status: "paid")
+
+    @order4 = Order.create!(user_id: @user.id,
+                            total_price: 39.30,
+                            items: [@item2],
+                            created_at: "2015-04-22 19:35:07",
+                            updated_at: "2015-04-23 10:22:17",
+                            status: "cancelled")
+
+    allow_any_instance_of(ApplicationController).to receive(:current_user) { @admin }
+
+    visit root_path
+    click_on "Order Dashboard"
+  end
+
+  xscenario 'can view specific order details' do
+    click_on "Order created on 06/09/15"
+
+    expect(page).to have_content "Order date/time: 2015-06-09 19:35:07"
+    expect(page).to have_content "Purchaser name: Sir Whats"
+    expect(page).to have_content "Purchaser email: fml@database.sucks"
+
+    expect(page).to have_content "Item: Eat Cheese and Die"
+    expect(page).to have_content "Quantity: 1"
+    expect(page).to have_content "Price: $50.65"
+    expect(page).to have_content "Item subtotal: $50.65"
+
+    expect(page).to have_content "Order total: $50.65"
+    expect(page).to have_content "Order status: paid"
+  end
+
+  xscenario 'can view item page through order details' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.